Dynatrace/dynatrace-operator
# Create an ActiveGate CR

## Description

We want to split out the ActiveGate section into its own CR, the first step is to just move what we currently have that is ActiveGate related into its own CR.

## How can this be tested?

- `make deploy` => check if activegate crd is in the cluster. 
- `make doc` => updated docs are in the corresponding folder

## Checklist
